src/CodeGen_X86.cpp

```diff
@@ -35,7 +35,7 @@ namespace {
 // i32(i16_a)*i32(i16_b) +/- i32(i16_c)*i32(i16_d) can be done by
 // interleaving a, c, and b, d, and then using pmaddwd. We
 // recognize it here, and implement it in the initial module.
-bool should_use_pmaddwd(Expr a, Expr b, vector<Expr> &result) {
+static bool should_use_pmaddwd(Expr a, Expr b, vector<Expr> &result) {
```

Comment:
nit, but 'static' should be effectively a no-op inside an anon namespace. curious as to why added.

Reply:
You're right. I probably missed the surrounding namespace with brackets.